cypress/utils/utils.ts

```diff
@@ -1949,3 +1949,19 @@ export function selectColumns(selectedColumns: string[], buttonText: string = sa
             clickByText(button, buttonText, true);
         });
 }
+export function getUniqueNamesMap<T extends { name: string }>(instanceArrays: T[][]) {
```

Comment:
Since this is a very specific method it would be nice to add a comment with information about what it does